fedmsg_meta_fedora_infrastructure/tests/pagure.py

```diff
@@ -1623,6 +1623,70 @@ class TestGitCommit(Base):
          'topic': 'io.pagure.prod.pagure.git.receive'
     }
 
+class TestIssueDrop(Base):
+    """ These messages are published when a ticket is closed against a
```

Comment:
OK, can we changed the subtitle to read ``yangl1996 deleted ticket docs-test#42: "foobar"'``?  :+1: pending that.